bin/aws-kinesis-agent-babysit

```diff
@@ -2,9 +2,10 @@
 
 # The cron-babysit periodically checks the status of the agent, and restart it if stopped abnormally
 
+PATH=/sbin:/usr/sbin:/bin:/usr/bin
```

Comment:
Shall we do PATH=/sbin:/usr/sbin:/bin:/usr/bin:$PATH in case we don't want it to be overridden?